object store admin docs improvements... (#1232)
galaxyproject/training-material: topics/admin/tutorials/object-store/tutorial.md

>    </object_store>
>    ```
>

> > 4. Add a `pre_task` to create the `/data2` folder, exactly like for the `/data` folder. The [file module](https://docs.ansible.com/ansible/latest/modules/file_module.html) for Ansible may be helpful.

Reviewer: ```suggestion
> 4. Add a `pre_task` to create the `/data2` folder [using the file module](https://docs.ansible.com/ansible/latest/modules/file_module.html), exactly like for the `/data` folder.
```
